### ISSUE TITLE
🐛 Bug: no way to save "false" for boolean field 

### ISSUE THREAD

**Issue by masciugo** [nocodb/nocodb#8473]
### Please confirm if bug report does NOT exist already ?

- [X] I confirm there is no existing issue for this

### Steps to reproduce ?

By providing a checkbox to save a boolean value there's no way to save the False value. If you leave it blank null will be saved. To do that you need to save True and then change it to False

<img width="1590" alt="Screenshot 2024-05-14 at 11 48 05" src="https://github.com/nocodb/nocodb/assets/454321/fc524cd3-6ab9-419e-8a36-a227fe7a6276">

### Desired Behavior

I d'like to specify False in the form

### Project Details

Node: **v20.11.1**
Arch: **arm64**
Platform: **linux**
Docker: **true**
RootDB: **sqlite3**
PackageVersion: **0.207.0**

**Comment by dstala** [comment]
@masciugo can you confirm if configuring default value (set & clear) helps?

**Comment by dstala** [comment]
With PG as rootDB - didn't see NULL being returned. It's either `true` or `false`

**Comment by masciugo** [comment]
> @masciugo can you confirm if configuring default value (set & clear) helps?

Yes I confirm but the problem is that I don't control the db structure . So I cannot change the default value

**Comment by masciugo** [comment]
> With PG as rootDB - didn't see NULL being returned. It's either `true` or `false`

We target mysql databases

**Comment by dstala** [comment]
some prefer to differentiate null & false. if you prefer false over null, we recommend using field edit modal in nocodb to set default value for the field (set it once and clear -- to mark it as false)

please reopen this ticket if that doesn't help
